pkg/client/meta/meta.go

```diff
@@ -111,6 +112,7 @@ func GetRegisteredSchema() *runtime.Scheme {
 		&v1alpha1.KogitoApp{}, &v1alpha1.KogitoAppList{},
 		&v1alpha1.KogitoDataIndex{}, &v1alpha1.KogitoDataIndexList{},
 		&v1alpha1.KogitoInfra{}, &v1alpha1.KogitoInfraList{})
+	s.AddKnownTypes(v1beta1.SchemeGroupVersion, &v1beta1.Kafka{}, &v1beta1.KafkaList{}, &v1beta1.KafkaTopic{}, &v1beta1.KafkaTopicList{})
```

Comment:
same here, please add `kafkav1beta1` since we have a lot of imports from other CRDs.

Reply:
done